go.mod

```diff
@@ -7,7 +7,7 @@ replace github.com/ipfs/go-verifcid => github.com/celestiaorg/go-verifcid v0.0.1
 require (
 	github.com/BurntSushi/toml v0.4.1
 	github.com/celestiaorg/celestia-core v0.0.2-0.20210924001615-488ac31b4b3c
-	github.com/celestiaorg/nmt v0.7.0
+	github.com/celestiaorg/nmt v0.7.1-0.20211110104120-56a05aec365f
```

Comment:
I think we should cut a release and use that instead. I'll do that and link the tag here.

Reply:
here: https://github.com/celestiaorg/nmt/releases/tag/v0.8.0